meson_options.txt

```diff
@@ -28,6 +28,7 @@ option('systemd', type : 'boolean', value : true, description : 'enable systemd
 option('systemd_root_prefix', type: 'string', value: '', description: 'Directory to base systemd’s installation directories on')
 option('elogind', type : 'boolean', value : false, description : 'enable elogind support')
 option('tests', type : 'boolean', value : true, description : 'enable tests')
+option('qt_tests', type: 'boolean', value: false, description: 'enable QT tests')
```

Comment:
Qt is spelled with lower case t.

Reply:
Thanks will fix it up.  I also need to relocated the code in the build system so that we don't have to add C++ compiler everywhere.